redwood-layout-api/src/commonMain/kotlin/app/cash/redwood/layout/api/properties.kt

```diff
@@ -15,17 +15,16 @@
  */
 @file:Suppress("MemberVisibilityCanBePrivate")
 
-package app.cash.redwood.layout
+package app.cash.redwood.layout.api
```

Comment:
Open to suggestions for a better name for this module/package. It's types shared by the schema, generated code, and widget implementations.

Reply:
`layout-api` sounds good to me :+1: